src/test/scala/is/hail/io/ImportGDBSuite.scala

```diff
@@ -124,15 +124,17 @@ class ImportGDBSuite extends SparkSuite {
     val gdbVariantSampleMatrix = LoadGDB(hc, reader, loader, query, workspace, arrName, ref)
     val vcfVariantSampleMatrix = LoadVCF(hc, reader, "src/test/resources/sample2.vcf")
 
+    val vcfVAInfo = vcfVariantSampleMatrix.vaSignature.query("info")
     val vcfAnn = vcfVariantSampleMatrix
       .rdd
-      .map(_._2._1.asInstanceOf[GenericRow].toSeq(3)) //map out the QUAL field, which is set to missing in GenomicsDB
+      .map { case (_, (va, _)) => vcfVAInfo(va) } //map out the QUAL field, which is set to missing in GenomicsDB
       .map(ann => ann.asInstanceOf[GenericRow].toSeq(2)) //TODO: this line maps to a field in the annotation that isn't affected by either bug (above) -- remove this map from tests after genomicsDB fixes issues
```

Comment:
if this is a field of the info struct, you should be able to add that to the query string above, a la `.query("info.FOO")`